driver/gl/raster_cache.go

```diff
@@ -0,0 +1,66 @@
+package gl
+
+import (
+	"image"
+	"os"
+	"sync"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+type rasterInfo struct {
+	pix     *image.RGBA
+	w, h    int
+	expires time.Time
+}
+
+var cacheDuration = (time.Minute * 5)
+var rasters = make(map[fyne.Resource]*rasterInfo)
+var rasterMutex sync.RWMutex
+
+func init() {
+	if t, err := time.ParseDuration(os.Getenv("FYNE_CACHE")); err == nil {
+		cacheDuration = t
+	}
+
+	go func() {
```

Comment:
I would personally prefer if we avoided starting goroutines within init. It might be better to do this on app startup.

Reply:
yes, it does have a bit of code smell doing it this way.  Admittedly, its a hack.

I think that the whole image cache concept should be a package anyway, and a reference to "the cache" should be stored against the app, and then accessed where its needed.

Trying to avoid going down that path yet, because the requirements for complete img/texture caching probably isnt that well understood yet. (At least in my head anyway)